docs/modules/getting-started/pages/editions.adoc

```diff
@@ -58,97 +58,14 @@ You can find more information on installing the Hazelcast editions in the follow
-[source,plain,subs="attributes+"]
-----
-├── LICENSE
-├── NOTICE
-├── bin
-│   ├── common.sh
-│   ├── hz-cli
-│   ├── hz-cli.bat
-│   ├── hz-cluster-admin
-│   ├── hz-cluster-cp-admin
-│   ├── hz-healthcheck
-│   ├── hz-start
-│   ├── hz-start.bat
-│   ├── hz-stop
-│   └── hz-stop.bat
-├── config
-│   ├── examples
-│   │   ├── hazelcast-client-full-example.xml
-│   │   ├── hazelcast-client-full-example.yaml
-│   │   ├── hazelcast-client.yaml
-│   │   ├── hazelcast-full-example.xml
-│   │   ├── hazelcast-full-example.yaml
-│   │   ├── hazelcast-security-hardened.yaml
-│   │   └── hazelcast.yaml
-│   ├── hazelcast-client.xml
-│   ├── hazelcast.xml
-│   ├── jmx_agent_config.yaml
-│   ├── jvm-client.options
-│   ├── jvm.options
-│   └── log4j2.properties
-├── custom-lib
-│   ├── hazelcast-3-connector-impl-{full-version}.jar
-│   ├── hazelcast-3.12.12.jar
-│   └── hazelcast-client-3.12.12.jar
-├── lib
-│   ├── cache-api-1.1.1.jar
-│   ├── hazelcast-3-connector-common-{full-version}.jar
-│   ├── hazelcast-3-connector-interface-{full-version}.jar
-│   ├── hazelcast-{full-version}.jar
-│   ├── hazelcast-download.properties
-│   ├── hazelcast-hibernate53-2.1.1.jar
-│   ├── hazelcast-jet-avro-{full-version}.jar
-│   ├── hazelcast-jet-cdc-debezium-{full-version}.jar
-│   ├── hazelcast-jet-cdc-mysql-{full-version}.jar
-│   ├── hazelcast-jet-cdc-postgres-{full-version}.jar
-│   ├── hazelcast-jet-csv-{full-version}.jar
-│   ├── hazelcast-jet-elasticsearch-7-{full-version}.jar
-│   ├── hazelcast-jet-files-azure-{full-version}.jar
-│   ├── hazelcast-jet-files-gcs-{full-version}.jar
-│   ├── hazelcast-jet-files-s3-{full-version}.jar
-│   ├── hazelcast-jet-grpc-{full-version}.jar
-│   ├── hazelcast-jet-hadoop-all-{full-version}.jar
-│   ├── hazelcast-jet-kafka-{full-version}.jar
-│   ├── hazelcast-jet-kinesis-{full-version}.jar
-│   ├── hazelcast-jet-protobuf-{full-version}.jar
-│   ├── hazelcast-jet-python-{full-version}.jar
-│   ├── hazelcast-jet-s3-{full-version}.jar
-│   ├── hazelcast-sql-{full-version}.jar
-│   ├── hazelcast-wm-4.0.jar
-│   ├── jansi-2.1.0.jar
-│   ├── jline-reader-3.19.0.jar
-│   ├── jline-terminal-3.19.0.jar
-│   ├── jline-terminal-jansi-3.19.0.jar
-│   ├── jmx_prometheus_javaagent-0.14.0.jar
-│   ├── log4j-api-2.14.0.jar
-│   ├── log4j-core-2.14.0.jar
-│   ├── log4j-slf4j-impl-2.14.0.jar
-│   ├── picocli-3.9.0.jar
-│   └── slf4j-api-1.7.30.jar
-└── licenses
-    ├── THIRD-PARTY.txt
-    ├── apache-v2-license.
-    ├── attribution.txt
-    └── hazelcast-community-license.txt
-├── management-center
-│   ├── ThirdPartyNotices.txt
-│   ├── bin
-│   │   ├── mc-conf.bat
-│   │   ├── mc-conf.sh
-│   │   ├── mc-start.cmd
-│   │   ├── mc-start.sh
-│   │   ├── start.bat
-│   │   ├── start.sh
-│   │   └── user-lib
-│   ├── hazelcast-management-center-{full-version}.jar
-│   └── license.txt
-└── release_notes.txt
-----
+- `bin` — utility scripts for application management
+// - `config` - application configuration files (including reference examples)
```

Comment:
What is the purpose of the double slash at the beginning? `//`

Reply:
It's a comment.

Reply:
I see, why is it commented out? I would expect it to be described here.

Reply:
I thought describing the config folder might be bordering on patronising… but I left it commented out in case of change of mind.

Reply:
> I would mention everything that's there, so when I do ls in the unzipped archive, this list and the contents are the same.

a683f95dc6ba968ae3c228779284c7f1d8ebbd1d